### PR TITLE
Bump SDK versions

### PR DESCRIPTION
**Pull request by york-wei**

### CODE REVIEW

File: plugin/plugin.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
         <source-file src="src/ios/CDVRadar.m"/>
         <podspec>
             <pods use-frameworks="true">
-                <pod name="RadarSDK" spec="3.5.9"/>
+                <pod name="RadarSDK" spec="3.9.7"/>
```
Comment (lmeier) [inline plugin/plugin.xml:59]: Same, let's get this to `3.9.12` -- will need:
- Exposes `user.failed_fraud` and `user.arrived_at_wrong_trip_destination` event types.
- Exposes `passed`, `inExclusionZone`, `inBufferZone`, and `distanceToBorder` on `RadarRegion`.
- Exposes `inaccurate` on `RadarFraud`.
- Improves the latency of `Radar.trackVerified()` calls.

**Review by lmeier — COMMENTED** [review]
dah, we should have shipped this earlier, that's my bad.

Let's get this to current native versions, there's a few fields to add @york-wei

**Comment by york-wei** [comment]
@lmeier bumped. Doesn't look like we need any code changes for these

**Review by lmeier — APPROVED** [review]
nice -- let's cut a release

**Pull request merged**